mvnverify.adoc

```diff
@@ -9,7 +9,7 @@
 ////
 === Running the tests
 
-If the server is still running from the previous steps, stop it first using the Maven `liberty:stop-server` goal:
+If the server is still running from the previous steps, stop it first using the Maven `liberty:stop-server` goal from command line in the `start` directory:
```

Comment:
How about..."If the server is still running from the previous steps, stop it with the Maven `liberty:stop-server` goal from the command line in the `start` directory:"

Reply:
I felt that "first" was already implied because it's the first step under this heading. :)